docs/en/stack/ml/anomaly-detection/ml-jobs-from-visuals.asciidoc

```diff
@@ -0,0 +1,58 @@
+[role="xpack"]
+[[ml-jobs-from-lens]]
+= Creating {anomaly-jobs} from Lens visualizations
+++++
+<titleabbrev>{anomaly-jobs-cap} from visualizations</titleabbrev>
+++++
+
+You can create {anomaly-jobs} from the compatible Lens charts on **Dashboard**. 
+
+[discrete]
+[[prereqs]]
+== Prerequisites
+
+* Only chart-like visualizations are compatible. Supported chart types are 
+`area`, `area_percentage_stacked`, `area_stacked`, `bar`, `bar_horizontal`, 
+`bar_horizontal_stacked`, `bar_percentage_stacked`, `bar_stacked`, and `line`.
+
+* The chart must contain a date field on one axis and it must be the same as the 
+default date field for the {data-source}.
+
+
+[discrete]
+[[create-job]]
+== Creating the job
+
+NOTE: You need to have a compatible visualization on **Dashboard** to create an 
+{anomaly-job}. If you don't have one but you want to try the feature out, go to 
+**Analytics > Dashboard** and select the `[Flight] Global Flight Dashboard` 
+which is based on the {kib} sample flight data set. Select the `Flight count` 
+visualization from the dashboard.
+
+
+. Go to **Analytics > Dashboard** and select a dashboard with a compatible 
+visualization.
+. Find the chart and select **More** from its **Options (...) menu**.
+. Select **Create {anomaly-job}**. The option is only displayed if the 
+visualization can be converted to an {anomaly-job} configuration.
+. (Optional) Select the layer from which the {anomaly-job} is created.
+
+[role="screenshot"]
+image::images/create-ad-job-from-lens.jpg[A screenshot of a chart with the Options menu opened]
+
+If the visualization has multiple compatible layers, you can select which layer 
+to use for creating the {anomaly-job}. If multiple fields are added to the chart or you  selected a `Break down by` field, the multi metric job wizard is used 
```

Comment:
Sorry, another nit pick.
The sentence about the vis layers and the multiple fields are unrelated and so shouldn't be in the same paragraph IMO.

The layer select sentence could possibly even have an accompanying picture.

Reply:
Good idea, thanks! I updated the content with a screenshot. It also separates the two sentences, so it's more evident that they are unrelated.